addons/mail/wizard/mail_template_preview_view.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <!-- Email Template Preview -->
+        <record model="ir.ui.view" id="mail_template_preview_form">
+            <field name="name">mail.template.preview.form</field>
+            <field name="model">mail.template.preview</field>
+            <field name="arch" type="xml">
+                <form string="Email Preview">
+                    <h3>Preview of <field name="mail_template_id" readonly="1" nolabel="1" class="oe_inline text-primary" options="{'no_open' : True}"/></h3>
+                    <div class="alert alert-danger" role="alert" attrs="{'invisible' : [('error_msg', '=', False)]}">
+                        <field name="error_msg" />
+                    </div>
+                    <div class="o_row">
+                        <span>Choose an example <field name="model_id" readonly="1"/> record:</span>
+                        <div>
+                            <field name="resource_ref" class="oe_inline" 
+                                options="{'hide_model': True, 'no_create': True, 'no_edit': True, 'no_open': True}"/>
+                        </div>
+                    </div>
+                    <div class="o_row">
+                        <span>Force the language: </span><field name="lang" class="oe_inline pr-2"/>
+                    </div>
+                    <group>
+                        <field name="subject" readonly="1"/>
+                        <field name="email_from" readonly="1"
+                            attrs="{'invisible':[('email_from','=', False)]}"/>
+                        <field name="partner_ids" widget="many2many_tags" readonly="1"
+                            attrs="{'invisible':[('partner_ids', '=', [])]}"/>
+                        <field name="email_to" readonly="1"
+                            attrs="{'invisible':[('email_to','=', False)]}"/>
+                        <field name="email_cc" readonly="1"
+                            attrs="{'invisible':[('email_cc','=', False)]}"/>
+                        <field name="reply_to" readonly="1"
+                            attrs="{'invisible':[('reply_to','=', False)]}"/>
+                        <field name="scheduled_date" readonly="1"
+                            attrs="{'invisible':[('scheduled_date','=', False)]}"/>
+                    </group>
+                    <field name="body_html" widget="html" readonly="1" nolabel="1" options='{"safe": True}'/>
+                    <field name="attachment_ids" widget="many2many_binary" readonly="1"/>
+                    <footer>
+                        <button string="Discard" class="btn-secondary" special="cancel"/>
+                    </footer>
+                </form>
+            </field>
+        </record>
+
+        <record id="wizard_mail_template_preview" model="ir.actions.act_window">
```

Comment:
Guidelines: `mail_template_preview_action`

Reply:
Thank you, change done!